.env-regtest-sample

```diff
@@ -70,4 +75,4 @@ SECURE_PATH=~/.sparkswap/secure
 # be applied and used
 #
 COMPOSE_PATH_SEPARATOR=:
-COMPOSE_FILE=docker-compose.yml:docker-compose.bitcoind.yml:docker-compose.litecoind.yml:docker-compose.override.yml
+COMPOSE_FILE=docker-compose.yml:docker-compose.bitcoind.yml:docker-compose.litecoind.yml:docker-compose.regtest.yml
```

Comment:
so no override file at all? Shouldn't we add it to the end of the path here?

Reply:
@treygriffith we no longer require an override file for our dev setup. If we add `docker-compose.override.yml` to this file AND we dont have an override file, youll receive an error:

```
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.override.yml'
```